app/controllers/application_controller.rb

```diff
@@ -11,10 +11,7 @@ def current_user
   def new_guest_user
   	Person.find(session[:current_user_id] = Person.create({}).id)
   end
-  def guest?
```

Comment:
None of the code cares if you are logged in.  I could put a conditional in the layout for which top navigation to use.